src/common/transformations/src/transformations/common_optimizations/align_mixed_fp32_fp16_types.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/align_mixed_fp32_fp16_types.hpp"
+
+#include <ngraph/rt_info.hpp>
+#include <openvino/opsets/opset1.hpp>
```

Comment:
I didn't find usage of opset1 in this file

Reply:
i will remove it

Reply:
done